tools/Dockerfiles/stylecheck

```diff
@@ -0,0 +1,22 @@
+FROM fedora:latest
```

Comment:
Do you wanna run the style check from rawhide -- some other unrelated packages might be broken? I'm not sure whether `latest` refers to 28 or 29 or rawhide.

Reply:
No, this script is only for checking style. `latest` refers to the latest release by Fedora project, so this is (currently) 28. See https://hub.docker.com/_/fedora/ for more information about Docker's Fedora images. Since our style checks don't really care about the version of the checkers, and it'd be nice to not update them every time a new release happens, I'm fine using `latest`. And no, hopefully 28 won't have broken packages and (hopefully) when 29 ships, it won't have broken packages either. :-)


`rawhide` is a different tag.